includes/class-wc-stripe-helper.php

```diff
@@ -209,10 +209,12 @@ public static function get_localized_messages() {
 				'card_declined'            => __( 'The card was declined.', 'woocommerce-gateway-stripe' ),
 				'missing'                  => __( 'There is no card on a customer that is being charged.', 'woocommerce-gateway-stripe' ),
 				'processing_error'         => __( 'An error occurred while processing the card.', 'woocommerce-gateway-stripe' ),
-				'invalid_request_error'    => __( 'Unable to process this payment, please try again or use alternative method.', 'woocommerce-gateway-stripe' ),
 				'invalid_sofort_country'   => __( 'The billing country is not accepted by SOFORT. Please try another country.', 'woocommerce-gateway-stripe' ),
 				'email_invalid'            => __( 'Invalid email address, please correct and try again.', 'woocommerce-gateway-stripe' ),
-			)
+				'invalid_request_error'    => is_add_payment_method_page()
+					? __( 'Unable to save this payment method, please try again or use alternative method.', 'woocommerce-gateway-stripe' )
+					: __( 'Unable to process this payment, please try again or use alternative method.', 'woocommerce-gateway-stripe' ),
+				)
```

Comment:
_Minor nitpick:_ I think the indentation here is 1 level too deep?

Reply:
Just the closing `)` that is, not the setup of the strings.

Reply:
Sorry, VSCode must've somehow auto-formatted this. Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1398/commits/b93335da34db318d40b9dc926f05b5134043d51c